includes/class-plugin.php

```diff
@@ -665,38 +666,68 @@ public function render_admin_bar( $wp_admin_bar ) {
             return;
         }
 
+        $html = <<<HTML
+            <style>
+                #wpadminbar ul li.redis-cache-error { background: #c00; }
+                #wpadminbar:not(.mobile) .ab-top-menu > li.redis-cache-error:hover > .ab-item { background: #b30000; color: #fff; }
+            </style>
+            <script>
+                document.querySelector('#wp-admin-bar-redis-cache-flush > a')
+                    .addEventListener('click', async function (event) {
```

Comment:
We have to use Vanilla js, because the `admin.js` wasn't loaded on the frontend.

Reply:
I'm wondering is it fine in WP mixing PHP and JS code? Maybe it would be better to use something like `wp_enqueue_script`?